pkg/didcomm/common/service/service.go

```diff
@@ -15,7 +15,7 @@ type InboundHandler interface {
 // OutboundHandler is handler for outbound messages
 type OutboundHandler interface {
 	// HandleOutbound handles outbound messages.
-	HandleOutbound(msg DIDCommMsg, myDID, theirDID string) error
+	HandleOutbound(msg DIDCommMsg, myDID, theirDID string) (string, error)
```

Comment:
would help to document the return values on this function. I see all implementations just returning an empty string?